### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by ashishHa**
Set up local machine and deployed to S3.

http://prajapathi.a.s3-website-us-east-1.amazonaws.com/

### CODE REVIEW

File: _assets/libs/bootstrap-4.0.0-alpha.6/Gemfile.lock [hunk 1]
```diff
@@ -1,58 +0,0 @@
-GEM
```
Comment (satyaprakash-k) [inline _assets/libs/bootstrap-4.0.0-alpha.6/Gemfile.lock]: Remove this file from libs folder.

**Comment by satyaprakash-k** [comment]
Add the changes done in milestone 1 too

File: _layouts/home.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
   {% include header.html %}
   <div class="container">
     <section class="middle">
-      <h1 class="display-5">{{ page.headline }}</h1>
```
Comment (satyaprakash-k) [inline _layouts/home.html]: use proper class name. class names, variable names, ID names all should have some significance with it's relative section/div.

**Comment by ashishHa** [comment]
created a new PR

**Pull request closed without merging**